localtileserver/widgets.py

```diff
@@ -14,8 +14,7 @@
 class LocalTileServerLayerMixin:
     """Mixin class for tile layers using localtileserver."""
 
-    # Prevent the client from being garbage collected
-    tile_server: TileClient
+    pass
```

Comment:
I don't think we want to remove this type annotation... any reason you chose to remove it?

Reply:
Oddly, the "pass" is present when using pip to install on Ubuntu 20.04 with python 3.10. Looks like its version 0.10.1 but github only shows version 0.10.0. I sometimes live life on the edge and make edits to code directly in .local/lib/python3.x/site-packages rather than cloning it